### PR TITLE
Update changelog, version, NOTICE.html

### PR DESCRIPTION
**Pull request by bwateratmsft**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-docker",
-    "version": "1.1.1-alpha",
+    "version": "1.2.0",
```
Comment (bwateratmsft) [inline package.json:3]: Still need to update `NOTICES.html` when #1934 is complete.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+### Added
+* Semantic highlighting support. [#1840](https://github.com/microsoft/vscode-docker/issues/1840)
+* Help and Feedback pane in explorer view. [#1893](https://github.com/microsoft/vscode-docker/issues/1893)
+* Docker Context pane in explorer view. [#1844](https://github.com/microsoft/vscode-docker/issues/1844)
+* Images can be pulled from the images list. [#1313](https://github.com/microsoft/vscode-docker/issues/1313)
+* Containers can be grouped by docker-compose project name. [#215](https://github.com/microsoft/vscode-docker/issues/215), [#1846](https://github.com/microsoft/vscode-docker/issues/1846)
+* A new setting, `docker.dockerodeOptions`, allowing any options to be provided to Dockerode. [#1459](https://github.com/microsoft/vscode-docker/issues/1459)
+
+### Changed
+* Any file named `Dockerfile.*` is now recognized as a Dockerfile. [#1907](https://github.com/microsoft/vscode-docker/issues/1907)
```
Comment (rcjsuen) [inline CHANGELOG.md:11]: What a time to be alive...

**Comment by bwateratmsft** [comment]
> Is it worth mentioning that 1.2.0 depends on Visual Studio Code 1.44? Or perhaps this should be something on the `README.md` file?

@rcjsuen Can't hurt! Though by the time we ship, probably about 95% will be upgraded.

File: CHANGELOG.md (hunk 2 above)
Comment (bwateratmsft) [reply CHANGELOG.md:11]: All this time we spent worrying if not-real-Dockerfiles like `Dockerfile.js` was a problem, and it ended up being 1/20th as common as real Dockerfiles with random extensions like `Dockerfile.amd64`.

**Pull request merged**